cmd/checks.go

```diff
@@ -42,3 +42,10 @@ func NewComponentCheck(name string) *ComponentCheck {
 		Name: name,
 	}
 }
+
+// NewExecutableCheck returns an intiialized instance of *ExecutableCheck
+func NewExecutableCheck(name string) *ExecutableCheck {
```

Comment:
Here and above, any reason not to have a return type of `DCOSChecker` instead of the concrete type (e.g. `*ExecutableCheck`?